Description: Fix the compatibility with the version of osgi-compendium in Debian
Author: Emmanuel Bourg <ebourg@apache.org>
Forwarded: not-needed
--- a/src/main/org/h2/util/OsgiDataSourceFactory.java
+++ b/src/main/org/h2/util/OsgiDataSourceFactory.java
@@ -288,7 +288,7 @@
      */
     static void registerService(BundleContext bundleContext,
             org.h2.Driver driver) {
-        Properties properties = new Properties();
+        java.util.Dictionary properties = new Properties();
         properties.put(
                 DataSourceFactory.OSGI_JDBC_DRIVER_CLASS,
                 org.h2.Driver.class.getName());
